lib/cogctl/actions/roles/util.ex

```diff
@@ -12,9 +12,17 @@ defmodule Cogctl.Actions.Roles.Util do
 
   def render(role_info, permission_info, group_info, sort) do
     Table.format(role_info, false)
-        <> "\n\nPermissions:\n" <> Table.format(permission_info, sort)
-        <> "\n\nGroups:\n" <> Table.format(group_info, sort)
+        <> "\n\nPermissions:\n" <> render_permissions(permission_info, sort)
+        <> "\n\nGroups:\n" <> render_groups(group_info, sort)
     |> display_output
   end
 
+  defp render_groups(group_info, sort) do
+    Table.format([["NAME", "ID"]] ++ group_info, sort)
```

Comment:
This now splits table creation across two modules. One creates the header, while the other, completely disconnected from the first, creates the table data, with nothing to ensure that the header and data match.